docs/directives/needbar.rst

```diff
@@ -4,6 +4,11 @@ needbar
 ========
 
 .. versionadded:: 0.7.5
+.. versionchanged:: nextversion
```

Comment:
Oh wait this is smack in the middle of some other docs isn't it?
Should probably move this block down a bit?